modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/search/spi/model/index/contributor/CommerceOrderModelDocumentContributor.java

```diff
@@ -58,10 +62,6 @@ public void contribute(Document document, CommerceOrder commerceOrder) {
 						"externalReferenceCode", "purchaseOrderNumber"
```

Comment:
Hey @smottal can you please double-check this? I'm unsure about the sorting of the first section of `Field` fields.